vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2736,16 +2745,12 @@ public void onRequestPermissionsResult(int aRequestCode, @NonNull String[] aPerm
      */
     private void manageSendMoreButtons() {
         boolean hasText = (mEditText.getText().length() > 0);
-        mSendImageView.setImageResource(hasText ? R.drawable.ic_material_send_green : R.drawable.ic_material_file);
-    }
-
-    /**
-     * Refresh the Account avatar
-     */
-    private void refreshSelfAvatar() {
-        // sanity check
-        if (null != mAvatarImageView) {
-            VectorUtils.loadUserAvatar(this, mSession, mAvatarImageView, mSession.getMyUser());
+        if (hasText == false) {
+            mStartCallIcon.setVisibility(View.VISIBLE);
```

Comment:
The `mStartCallIcon` visibility must follow the same constraints than `mStartCallLayout`
(see existing refreshCallButtons methods).

I think we should keep the existing item mStartCallLayout (id:room_start_call_image_view) to handle correctly the call icon**s**,  by updating only the start call icon.
The send message icon should be handled by the existing "mSendImageView" by renaming it, and removing the attachment option from it...